reqs/requirements-dev-contrib.txt

```diff
@@ -4,8 +4,4 @@ invoke>=1.7.1
 isort==5.10.1
 mypy==0.991
 pre-commit>=2.6.0
-pydantic>=1.0,<2.0
-pytest-cov>=2.8.1
-pytest-order>=0.9.5
-pytest-random-order>=1.0.4
```

Comment:
I don't know why these pytest plugins were even in this file. Seems like they don't belong here.

I don't know why they really belong in `reqs/requirements-dev-lite.txt` either but at least that file has other testing dependencies.
🤷

Reply:
Is `reqs/requirements-dev-contrib.txt` used in other pipelines? Looks like `azure-pipelines-contrib.yml` in the linting stage - are any of these dependencies required for that? (I don't think so but haven't looked deeply).

Reply:
Yes, but the `linting` step doesn't need these `pytest` plugins. I think having them in this file in the first place may have been a mistake.

Reply:
I think that the dev-lite was meant to be the least amount of packages needed to run tests locally. Those plugins aren't needed for a local run, so they were added to the contrib file

Reply:
@kenwade4 Thanks for the explanation. That makes sense.